src/map/pc.c

```diff
@@ -12440,6 +12453,9 @@ void pc_defaults(void) {
 	pc->getmaxspiritball = pc_getmaxspiritball;
 
```

Comment:
missing init code for ``pc->dbs``

Reply:
missing init code for introduced vectors. For each VECTOR_DECL need call VECTOR_INIT here